frb_example/benches/dart/lib/bridge_definitions.dart

```diff
@@ -0,0 +1,138 @@
+
+  Future<int> handleU32({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleU32ConstMeta;
+
+  Future<int> handleU64({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleU64ConstMeta;
+
+  Future<int> handleI8({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleI8ConstMeta;
+
+  Future<int> handleI16({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleI16ConstMeta;
+
+  Future<int> handleI32({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleI32ConstMeta;
+
+  Future<int> handleI64({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleI64ConstMeta;
+
+  Future<double> handleF32({required double input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleF32ConstMeta;
+
+  Future<double> handleF64({required double input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleF64ConstMeta;
+
+  Future<String> handleString({required String input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleStringConstMeta;
+
+  bool handleSyncBool({required bool input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleSyncBoolConstMeta;
+
+  int handleSyncU32({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleSyncU32ConstMeta;
+
+  int handleSyncU64({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleSyncU64ConstMeta;
+
+  int handleSyncI8({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleSyncI8ConstMeta;
+
+  int handleSyncI16({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleSyncI16ConstMeta;
+
+  int handleSyncI32({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleSyncI32ConstMeta;
+
+  int handleSyncI64({required int input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleSyncI64ConstMeta;
+
+  double handleSyncF32({required double input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleSyncF32ConstMeta;
+
+  double handleSyncF64({required double input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleSyncF64ConstMeta;
+
+  String handleSyncString({required String input, dynamic hint});
+
+  FlutterRustBridgeTaskConstMeta get kHandleSyncStringConstMeta;
+}
+
+/// metric used for [continuous-benchmark](https://github.com/marketplace/actions/continuous-benchmark) worflow
+class Metric {
+  final String name;
+  final int? value;
+
+  /// benchmarking time unit is platform-dependent
+  final Unit unit;
+
+  /// allows to provide extra context for benchmarked wired function call
+  /// e.g. `reverse 1,000 uuids`
+  ///
+  /// `non-final` to allow setting Rust metric `extra` from Dart
+  ///
+  /// TODO: once `hint` can be sent and consumed on Rust side, this field can be `final`
+  String? extra;
```

Comment:
Or, maybe we can create a new class:

```
class MetricClassOnTheDartSide {
  final String name; value; unit; ...;
  final whatever extra; // FINAL here
}
```

Reply:
It's better to have Rust + FRB generates the `Metric`, isn't it ? I don't really see any benefit here.

Reply:
Anyway this is just nits :)

Reply:
My 2 cents on this is, if you allow me: as long as `Metric` is declared in `api.rs` there's no maintenance cost since it gets codegened. But if you add an extra `MetricClassOnTheDartSide`, then you need to remember to maintain it alongside its `Metric` counterpart.

The real fix here, IMHO, is to provide a way to pipe `hint` from Dart down to Rust.
Then this `extra` field can just be `final`.